tests/collection/collection-layout.spec.ts

```diff
@@ -0,0 +1,157 @@
+    await collectionPage.infiniteScroller.clickFirstResultAndCheckRedirectToDetailsPage();
+  });
+});
+
+test(`Sort by All-time views in Tile view`, async ({ collectionPage }) => {
+  await test.step('Switch to tile view mode', async () => {
+    await collectionPage.infiniteScroller.clickViewMode(
+      LayoutViewModeLocator.TILE,
+    );
+  });
+
+  await test.step('Sort by All-time views - descending order', async () => {
+    await collectionPage.sortBar.applySortFilter('All-time views');
+    await collectionPage.sortBar.clickSortDirection('descending');
+  });
+
+  await test.step('Check the first 10 results if sort filters were applied', async () => {
+    await collectionPage.infiniteScroller.checkSortingResults(
+      'All-time views',
+      'descending',
+      10,
+    );
+  });
+
+  await test.step('Check if URL changed with correct sort filter and sort order param', async () => {
+    await collectionPage.searchPage.checkURLParamsWithSortFilter(
+      'All-time views',
+      'descending',
+    );
+  });
+});
+
+test(`Sort by Date published in List view`, async ({ collectionPage }) => {
+  await test.step('Switch to list view mode', async () => {
+    await collectionPage.infiniteScroller.clickViewMode(
+      LayoutViewModeLocator.LIST,
+    );
+  });
+
+  await test.step('Sort by Date published - descending order', async () => {
+    await collectionPage.sortBar.applySortFilter('Date published');
+    await collectionPage.sortBar.clickSortDirection('descending');
+  });
+
+  await test.step('Check the first 10 results if sort filters were applied', async () => {
+    await collectionPage.infiniteScroller.checkSortingResults(
+      'Date published',
+      'descending',
+      10,
+    );
+  });
+
+  await test.step('Check if URL changed with correct sort filter and sort order param', async () => {
+    await collectionPage.searchPage.checkURLParamsWithSortFilter(
+      'Date published',
+      'descending',
+    );
+  });
+});
+
+test(`Sort by Date archived (ascending) in Compact view`, async ({
+  collectionPage,
+}) => {
+  await test.step('Switch to compact view mode', async () => {
+    await collectionPage.infiniteScroller.clickViewMode(
+      LayoutViewModeLocator.COMPACT,
+    );
+  });
+
+  await test.step('Sort by Date archived - ascending order', async () => {
+    await collectionPage.sortBar.applySortFilter('Date archived');
+    await collectionPage.sortBar.clickSortDirection('ascending');
+  });
+
+  await test.step('Check list column headers for sort filter', async () => {
+    await collectionPage.searchPage.checkCompactViewModeListLineDateHeaders(
+      'Date archived',
+    );
+  });
+
+  await test.step('Check if URL changed with correct sort filter and sort order param', async () => {
+    await collectionPage.searchPage.checkURLParamsWithSortFilter(
+      'Date archived',
+      'ascending',
+    );
+  });
+});
```

Comment:
please look at these test blocks in all spec.ts files and see if we can collapse some down.. it would read better and showcase the tests in a way that clearly shows what we are testing

Reply:
updated